docs/multicluster/quick-start.md

```diff
@@ -114,7 +114,8 @@ Secret manifest and other ClusterSet join arguments to file `join-config.yml`
 command (with the `--config-file` option) to join the ClusterSet with these
 arguments. If you want to use a separate token for each member cluster for
 security considerations, you can run the following commands to create a token
-and use the token to join the ClusterSet:
+and use the token with the generated config file `join-config.yml` to join
```

Comment:
Probably: "use the token (together with the previously generated configuration file `join-config.yml`)"